boards/DCM/Src/App/states/App_SharedStates.c

```diff
@@ -31,3 +35,62 @@ void App_SharedStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     App_CanTx_SetPeriodicSignal_LEFT_INVERTER_SWITCH(
         can_tx, App_InverterSwitches_IsLeftOn(inverter_switches));
 }
+
+void App_SharedStates_ConfigInverterSwitches(
+    const struct DcmCanRxInterface *can_rx,
+    struct InverterSwitches *       inv_switches)
+{
+    if (App_CanRx_INV_LOAD_SWITCHES_CMD_GetSignal_CONFIG_LEFT_SWITCH(can_rx))
+    {
+        App_InverterSwitches_TurnOnLeft(inv_switches);
+    }
+    else
+    {
+        App_InverterSwitches_TurnOffLeft(inv_switches);
+    }
+
+    if (App_CanRx_INV_LOAD_SWITCHES_CMD_GetSignal_CONFIG_RIGHT_SWITCH(can_rx))
+    {
+        App_InverterSwitches_TurnOnRight(inv_switches);
+    }
+    else
+    {
+        App_InverterSwitches_TurnOffRight(inv_switches);
+    }
+}
+
+void App_SharedStates_HandleClearInvFaultsCmd(
+    const struct DcmCanTxInterface *can_tx,
+    struct DcmCanRxInterface *      can_rx)
+{
+    if (App_CanRx_CLEAR_INV_FAULT_CMD_GetSignal_CLEAR_LEFT_FAULT(can_rx))
+    {
+        const struct CanMsgs_dcm_invl_read_write_param_command_t
+            invl_cmd_msg = { .d1_parameter_address_command =
```

Comment:
perhaps we could use a #define for the contents of this CAN message considering it is the same payload for both INVL and INVR

Reply:
done